packages/ses/src/module-instance.js

```diff
@@ -0,0 +1,314 @@
+import { performEval } from './evaluate.js';
+import { getCurrentRealmRec } from './realm-rec.js';
+
+// eslint-disable-next-line no-undef
+const harden = (typeof SES !== 'undefined' && SES.harden) || Object.freeze;
```

Comment:
Given the layering, I think I might just reduce this to freeze. SES makes harden and it will appear as globalThis.harden after lockdown, but not before.